gp-templates/project-delete.php

```diff
@@ -0,0 +1,16 @@
+<?php
+gp_title( sprintf( __( 'Delete Project %s &lt; GlotPress', 'glotpress' ),  $project->name ) );
+gp_breadcrumb_project( $project );
+gp_tmpl_header();
+?>
+<h2><?php echo wptexturize( sprintf( __( 'Delete project "%s"', 'glotpress' ), esc_html( $project->name ) ) ); ?></h2>
```

Comment:
Copied straight from the edit template:

https://github.com/GlotPress/GlotPress-WP/blob/develop/gp-templates/project-edit.php#L6

Is it better to remain consistent and then create a PR to solve all of them or update this one instance?